docs/source/global_surface_emission.ipynb

```diff
@@ -992,7 +992,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You now need to reinstalled the package after replacing the contents of ``xpsi/surface_radiation_field/local_variables.pyx`` with the exact contents of ``xpsi/surface_radiation_field/archive/tutorial_spot_and_ring.pyx``. The ``local_variables.pyx`` extension module must transform some set of *global* variables into *local* variables at the spacetime event defined by the intersection of a ray with the stellar surface. A vector of local variables is then passed to the ``xpsi/surface_radiation_field/hot.pyx`` module for evaluation of the specific intensity of radiation, w.r.t a local comoving surface frame, that after Lorentz transformation is transported along the ray to the image plane.\n",
+    "You now need to reinstalled the package after replacing the contents of ``xpsi/surface_radiation_field/local_variables.pyx`` with the exact contents of ``xpsi/surface_radiation_field/archive/local_variables/tutorial_spot_and_ring.pyx``. The ``local_variables.pyx`` extension module must transform some set of *global* variables into *local* variables at the spacetime event defined by the intersection of a ray with the stellar surface. A vector of local variables is then passed to the ``xpsi/surface_radiation_field/hot.pyx`` module for evaluation of the specific intensity of radiation, w.r.t a local comoving surface frame, that after Lorentz transformation is transported along the ray to the image plane.\n",
```

Comment:
could you also fix my typo "reinstalled" please?

Reply:
Can you double check if it went through? I pushed the change on my fit_branch and clicked for the pull request from here, but I cannot see the commit message anywhere, so I wondered if it worked...

Reply:
yes, it worked, thanks!